### PR TITLE
feat: Require inbound cidrs be set explicitly

### PR DESCRIPTION
**Pull request by gls4**
This PR  **requires** that a value be **explicitly** provided for `allowed_inbound_cidrs`. To allow all inbound connections, the value `["*"]` should be used; this is the value set in the root of the module. To restrict access to specific IPs and/or IP ranges, use CIDR notation:
```
allowed_inbound_cidrs = [ "x.x.x.x/32", "x.x.x.x/24" ]
```

This change is also reflected in the examples.

### CODE REVIEW

File: modules/app_lb/url_map/variables.tf [hunk 1]
```diff
@@ -1,28 +1,31 @@
-variable "namespace" {
-  type        = string
-  description = "Friendly name prefix used for tagging and naming AWS resources."
+variable "allowed_inbound_cidrs" {
+  description = "(Optional) Allow HTTP(S) traffic to W&B. Defaults to all connections."
```
Comment (yogeshg) [inline modules/app_lb/url_map/variables.tf:2]: Sorry, Is it optional or not now?

Comment (gls4) [reply modules/app_lb/url_map/variables.tf]: Oh, good catch. Amended. :-)

**Review by flamarion — DISMISSED** [review]
This change will block the access but will still create the public load balancer.
I believe we should suppress the public load balancer creation when setting an env var like `internal_something` for example.

**Review by elainaRenee — APPROVED** [review]

**Review by wandb-kc — APPROVED** [review]

**Comment by jsbroks** [comment]
This PR is included in version 1.15.0 :tada:

**Pull request merged**